submissions/Chimoney-Python/pychimoney/Chimoney.py

```diff
@@ -0,0 +1,27 @@
+from pychimoney import Info, Account, Payouts, SubAccount, Wallet, Redeem, BaseAPI
+
+
+class Chimoney(BaseAPI):
+    """
+    Root API
+
+    This is the root API for the Chi Money API.
+    """
+
+    def __init__(self):
+        super().__init__()
```

Comment:
You are not passing the `auth_key` to the `BaseAPI` or collecting it from the `Chimoney` constructor  here so I doubt it'll be available for use in other classes. Is there another place where the instantiation is happening?

Reply:
Fixed and Push.

Planning to change the implementation of the Initailization class in a later update.

Reply:
> You are not passing the `auth_key` to the `BaseAPI` or collecting it from the `Chimoney` constructor here so I doubt it'll be available for use in other classes. Is there another place where the instantiation is happening?

Was a mistake, seems I wasn't looking at what I implemented at the time